spec/controllers/collection_items_controller_spec.rb

```diff
@@ -15,7 +29,63 @@
       end
     end
 
-    context "create" do
+    context "rejected parameter for collection with items in" do
+      let(:owner) { @collection.owners.first.user }
+
+      it "includes rejected items" do
+        fake_login_known_user(owner)
+        get :index, collection_id: @collection.name, rejected: true
+        expect(response).to have_http_status(:success)
+        expect(assigns(:collection_items)).to include @rejected_work_item
+      end
+
+      it "excludes approved and invited items" do
+        fake_login_known_user(owner)
+        get :index, collection_id: @collection.name, rejected: true
+        expect(assigns(:collection_items)).not_to include @approved_work_item
+        expect(assigns(:collection_items)).not_to include @invited_work_item
+      end
+    end
+
+    context "invited parameter for collection with items in" do
+      let(:owner) { @collection.owners.first.user }
+
```

Comment:
Not super important, but one way to reduce the repetition here might be to take these  two lines and make them into a before block:
```
fake_login_known_user(owner)
get :index, collection_id: @collection.name, invited: true
```
The same would also apply in the next context with
```
 fake_login_known_user(owner)
get :index, collection_id: @collection.name, approved: true
```
However, it's only a tiny bit of repetition.